src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaWebsiteControllerTest.php

```diff
@@ -233,7 +233,7 @@ public function testResponseHeader()
         $date = new DateTime();
         $date->modify('+1 month');
 
-        $this->client->request(
+        $this->client->jsonRequest(
```

Comment:
seems like i forgot this. this should be a normal request as it is no json api:

```suggestion
        $this->client->request(
```